morpheus/_lib/src/python_modules/messages.cpp

```diff
@@ -110,10 +110,17 @@ PYBIND11_MODULE(messages, m)
     mrc::node::EdgeConnector<std::shared_ptr<morpheus::MultiResponseProbsMessage>,
                              std::shared_ptr<morpheus::MultiMessage>>::register_converter();
 
+    py::class_<MutableCtxMgr, std::shared_ptr<MutableCtxMgr>>(m, "MutableCtxMgr")
+        .def("__enter__", &MutableCtxMgr::enter, py::return_value_policy::reference)
+        .def("__exit__", &MutableCtxMgr::exit)
+        .def_property_readonly("df", &MutableCtxMgr::df_property, py::return_value_policy::reference);
```

Comment:
We should discuss, but I feel like these methods could be put on a different class or return different object. Something like the following would be nice:
```python
with meta.mutable_dataframe() as df:
    df["col"] = 4
```
Where the context manager `__enter__` returns the dataframe object directly.